scripts/deploy.js

```diff
@@ -24,28 +23,22 @@ function upload() {
     .uploadExisting(extesnionSource)
     .then((res) => {
       console.log('Successfully uploaded the ZIP');
-
-      // call publish API on success
+      webStore
+      .publish()
+      .then((res) => {
+        console.log('Successfully published the newer version');
+      })
+      .catch((error) => {
+        console.log(`Error while publishing uploaded extension: ${error}`);
+        process.exit(1);
+      });
     })
     .catch((error) => {
       console.log(`Error while uploading ZIP: ${error}`);
       process.exit(1);
     });
 }
 
-function publish() {
-  // publish the uploaded zip
-  webStore
-    .publish()
-    .then((res) => {
-      console.log('Successfully published the newer version');
-    })
-    .catch((error) => {
-      console.log(`Error while publishing uploaded extension: ${error}`);
-      process.exit(1);
-    });
-}
-
 zipFolder(folderName, zipName, function (err) {
```

Comment:
Hey, @wxharry, since we have `zip release/hypercrx.zip -r build/*` in the "Build" step, I think zipFolder is not needed here.